modules/tensor_mechanics/src/materials/TempDepHardeningStressUpdate.C

```diff
@@ -65,7 +65,7 @@ RecomputeRadialReturnTempDepHardening::RecomputeRadialReturnTempDepHardening(con
 }
 
 void
-RecomputeRadialReturnTempDepHardening::computeStressInitialize(Real effectiveTrialStress)
+TempDepHardeningStressUpdate::computeStressInitialize(Real effectiveTrialStress)
```

Comment:
Not a fan of `TempDep`. Why not `TemperatureDependent`? Or is it `TemporaryDeposit`?